gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-bin.zip
```

Comment:
Plugin `com.gradleup.shadow` requires at least Gradle 8.3, so `gradlew` should provide this. We could also move to the latest Gradle 8 version (v8.10.2) if you like.

Reply:
Hi @EnricoMi I believe one of the other open PRs (or maybe even approved and closed by now) is related to upgrading Gradle to v8. In that case, we may be duplicating the change here. 
Can you please check and confirm the same ?

Reply:
I have merged master with conflicts and could find any open PR that touches this file.